tap_github/repository_streams.py

```diff
@@ -891,7 +891,16 @@ def get_records(self, context: Optional[Dict] = None) -> Iterable[Dict[str, Any]
             self.logger.debug(f"No comments detected. Skipping '{self.name}' sync.")
             return []
 
-        return super().get_records(context)
+        # Getting issue comments is sometimes too expensive on large repos and results server errors.
+        # To avoid crashing the tap completely, we wrap the get_records in a try/except.
+        try:
+            return super().get_records(context)
+        except ValueError:
+            self.logger.warning(
```

Comment:
Is it worth filtering on the error message itself (if at all possible), so as to ignore only these specific errors, and not ones we actually want to crash on?

Reply:
I added a string check, not sure if it's the best way though